site/ko/alpha/tutorials/text/text_generation.ipynb

```diff
@@ -0,0 +1,1885 @@
+     "height": 530.0
+    },
+    "colab_type": "code",
+    "executionInfo": {
+     "elapsed": 49082.0,
+     "status": "ok",
+     "timestamp": 1.551671325218E12
+    },
+    "id": "yG_n40gFzf9s",
+    "outputId": "8ce923b9-ccaa-45f4-a8dc-5f53f8a0495a"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Collecting tensorflow-gpu==2.0.0-alpha0\n",
+      "Successfully installed google-pasta-0.1.4 tb-nightly-1.14.0a20190303 tensorflow-estimator-2.0-preview-1.14.0.dev2019030300 tensorflow-gpu==2.0.0-alpha0-2.0.0.dev20190303\n"
+     ]
+    }
+   ],
+   "source": [
+    "from __future__ import absolute_import, division, print_function, unicode_literals\n",
+    "\n",
+    "!pip install tensorflow-gpu==2.0.0-alpha0\n",
+    "import tensorflow as tf\n",
+    "\n",
+    "import numpy as np\n",
+    "import os\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "EHDoRoc5PKWz"
+   },
+   "source": [
+    "### 셰익스피어 데이터 셋 다운로드\n",
+    "\n",
+    "본인의 데이터로 변경하여 다음 코드를 실행하세요."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 51.0
+    },
+    "colab_type": "code",
+    "executionInfo": {
+     "elapsed": 49517.0,
+     "status": "ok",
+     "timestamp": 1.551671325654E12
+    },
+    "id": "pD_55cOxLkAb",
+    "outputId": "db08f260-286d-4476-f78e-8926ecd05cb3"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Downloading data from https://storage.googleapis.com/download.tensorflow.org/data/shakespeare.txt\n",
+      "1122304/1115394 [==============================] - 0s 0us/step\n"
+     ]
+    }
+   ],
+   "source": [
+    "path_to_file = tf.keras.utils.get_file('shakespeare.txt', 'https://storage.googleapis.com/download.tensorflow.org/data/shakespeare.txt')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "UHjdCjDuSvX_"
+   },
+   "source": [
+    "### 데이터 읽기\n",
+    "\n",
+    "먼저, 텍스트를 살펴봅시다:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 34.0
+    },
+    "colab_type": "code",
+    "executionInfo": {
+     "elapsed": 49517.0,
+     "status": "ok",
+     "timestamp": 1.551671325655E12
+    },
+    "id": "aavnuByVymwK",
+    "outputId": "277c27cb-3dfb-4b05-f682-11d4db376736"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Length of text: 1115394 characters\n"
+     ]
+    }
+   ],
+   "source": [
+    "# 읽고, py2 compat에 대해 디코딩합니다.\n",
+    "text = open(path_to_file, 'rb').read().decode(encoding='utf-8')\n",
+    "# 텍스트의 길이는 그 안에있는 문자의 수입니다.\n",
+    "print ('Length of text: {} characters'.format(len(text)))"
```

Comment:
There is no reason that not translate sentences which is in print function  
So I suggest translating 'Length of text: {} characters' :)